dataflow/src/test/java/reachDefs/ReachDefinitions.java

```diff
@@ -0,0 +1,43 @@
+package reachDefs;
+
+import org.checkerframework.dataflow.analysis.ForwardAnalysis;
+import org.checkerframework.dataflow.analysis.ForwardAnalysisImpl;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeLauncher;
+import org.checkerframework.dataflow.reachdefinitions.ReachDefinitionsStore;
+import org.checkerframework.dataflow.reachdefinitions.ReachDefinitionsTransfer;
+import org.checkerframework.dataflow.reachdefinitions.ReachDefinitionsValue;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+/** Used in liveVariableTest Gradle task to test the LiveVariable analysis. */
+public class ReachDefinitions {
+
+    /**
+     * The main method expects to be run in dataflow/tests/live-variable directory.
+     *
+     * @param args not used
+     */
+    public static void main(String[] args) {
+
+        String inputFile = "Test.java";
```

Comment:
A lot of this code is the same in the live variable demo. Can you extract a common test file that reduces this duplication?

Reply:
Got it, will do it with a new pr.

Reply:
@al3xliu Ping on this. Having this extracted would be good for both new analysis PRs.